src/components/NavigationGroup/NavigationGroup.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { INavigationGroupProps } from './navigationGroup.Props';
+import { autobind } from '../../utilities/autobind';
+import './NavigationGroup.scss';
+import * as classNames from 'classnames';
+
+export class NavigationGroup extends React.Component<INavigationGroupProps, void> {
```

Comment:
It would be better if navigation group extends React.PureComponent, as it will automatically compare passed props.